tools/code-analyzer/src/commands/analyzer/index.ts

```diff
@@ -81,49 +87,28 @@ export default class Analyzer extends Command {
 	async run(): Promise< void > {
 		const { args, flags } = await this.parse( Analyzer );
 
-		this.validateArgs( flags.source );
-
-		const patchContent = generatePatch(
+		const diff = await generateDiff(
 			flags.source,
-			args.compare,
 			flags.base,
-			( e: string ): void => this.error( e )
+			args.compare,
+			this.error
 		);
 
 		const pluginData = this.getPluginData( flags.plugin );
 		this.log( `${ pluginData[ 1 ] } Version: ${ pluginData[ 0 ] }` );
 
 		// Run schema diffs only in the monorepo.
-		if ( flags.source === 'woocommerce/woocommerce' ) {
+		if ( flags.is_woocommerce ) {
 			const schemaDiff = await generateSchemaDiff(
-				flags.source,
+				'woocommerce/woocommerce',
```

Comment:
Passing fixed args to backward support this command until we migrate this code as well.